packages/go_router/lib/src/configuration.dart

```diff
@@ -375,22 +394,22 @@ class RouteConfiguration {
     return const <RouteMatchBase>[];
   }
 
-  /// Processes redirects by returning a new [RouteMatchList] representing the new
-  /// location.
+  /// Processes route-level redirects by returning a new [RouteMatchList] representing the new
+  /// location. This method now handles ONLY route-level redirects.
```

Comment:
Our styleguide suggest first paragraph must be a brief one sentence describe the functionality. other supplementary doc needs to be another paragraph with a new line in between